doc/manual/expressions/language-operators.xml

```diff
@@ -32,7 +32,9 @@ weakest binding).</para>
         <replaceable>e</replaceable>.  (An attribute path is a
         dot-separated list of attribute names.)  If the attribute
         doesn’t exist, return <replaceable>def</replaceable> if
-        provided, otherwise abort evaluation.</entry>
+        provided, otherwise abort evaluation. A function equivalent to this
+        operator, <literal>getAttrFromPath</literal>, is available in the
+        <literal>nixpkgs</literal> library.</entry>
```

Comment:
Aren't `builtins.getAttr` and `builtins.hasAttr slighty different as they cannot handle attribute paths?

```
> let s = { a = { b = 1; }; }; in  builtins.hasAttr "a.b" s
false
```

and

```
> let s = { a = { b = 1; }; }; in  builtins.getAttr "a.b" s
error: attribute ‘a.b’ missing, at (string):1:34
```